src/main/java/org/sagebionetworks/web/client/presenter/RejectReasonWidget.java

```diff
@@ -0,0 +1,151 @@
+
+    // Common reasons for user rejection
+    public static String REJECT_TAKE_SYNAPSE_QZ = "Take and pass the Synapse Certification quiz. ";
+    public static String REJECT_ADD_INFO = "Add at least one piece of information (education, employment, etc.) to your ORCID profile and set it to “public. ";
+    public static String REJECT_INITIAL_BOX = "Add at least one piece of information (education, employment, etc.) to your ORCID profile and set it to “public. ";
+    public static String REJECT_SUBMIT_DOCS = "Submit an accepted form of identity attestation documentation (e.g., letter from a signing official). ";
+    public static String REJECT_CUSTOM_REASON = "custom text to insert";
+
+    private String userName;
+    private RejectReasonView view;
+    UserProfileAsyncHandler handler;
+    CallbackP<String> callback;
+
+    @Inject
+    public RejectReasonWidget (UserProfileAsyncHandler handler, RejectReasonView view) {
+        this.handler = handler;
+        this.view = view;
+        this.view.setPresenter(this);
+        this.userName = "";
+        this.callback = null;
+    }
+
+    private void setUserProfileName(UserProfile user) {
+        this.userName = DisplayUtils.getDisplayName(user);
+    }
+
+    /*
+        For testing purposes only relay the userName to assert
+        that its being properly found.
+     */
+    public String getUserName() {
+        return this.userName;
+    }
+
+    // TODO: Remove when feature is in prod, used for demo purposes in ComingSoon
+    public void setUserProfileName(String user) {
+        this.userName = user;
+    }
+
+    public void show(String userID, CallbackP<String> callback) {
+        handler.getUserProfile(userID, new AsyncCallback<UserProfile>() {
+            @Override
+            public void onFailure(Throwable throwable) {
+                view.showError("Could not find user id -- " + userID + "\nError -- " +  throwable.getMessage());
+                view.show();
+            }
+
+            @Override
+            public void onSuccess(UserProfile user) {
+                setUserProfileName(user);
+                view.show();
+            }
+        });
+        this.callback = callback;
+    }
+
+    // TODO: Remove when feature is in prod, used for demo purposes in ComingSoon
+    public void show(String userID) {
+        handler.getUserProfile(userID, new AsyncCallback<UserProfile>() {
+            @Override
+            public void onFailure(Throwable throwable) {
+                setUserProfileName("John Doe");
+                view.show();
+            }
+
+            @Override
+            public void onSuccess(UserProfile user) {
+                setUserProfileName(user);
+                view.show();
+            }
+        });
+    }
+
+    @Override
+    public void updateResponse() {
+        String output = "";
+
+        if (view.optionOneIsUsed()) {
```

Comment:
I think the standard java naming convention for boolean property method getters is the form:
`is<PropertyName>`
Not important, just another note.